src/sliding_sync.rs

```diff
@@ -565,7 +576,28 @@ async fn async_worker(
                     }
                 });
             }
+            MatrixRequest::GetNumberUnreadMessages { room_id } => {
+                let sender = {
+                    let mut all_room_info = ALL_ROOM_INFO.lock().unwrap();
+                    let Some(room_info) = all_room_info.get_mut(&room_id) else {
+                        log!("Skipping pagination request for not-yet-known room {room_id}");
```

Comment:
fix copy-paste error message here
```suggestion
                        log!("Skipping get number of unread messages request for not-yet-known room {room_id}");
```